lua/hbac/utils.lua

```diff
@@ -0,0 +1,14 @@
+M = {}
+
+M.get_buffers = function()
+	return vim.tbl_filter(function(bufnr)
+		local name = vim.api.nvim_buf_get_name(bufnr)
+		local listed = vim.api.nvim_buf_get_option(bufnr, "buflisted")
+		if name ~= "" and listed then
+			return true
+		end
+		return false
+	end, vim.api.nvim_list_bufs())
```

Comment:
Would you mind if I merge this first?
https://github.com/axkirillov/hbac.nvim/pull/8

Then this becomes
```lua
			local buffers = vim.tbl_filter(function(buf)
				return vim.api.nvim_buf_get_option(buf, "buflisted")
			end, vim.api.nvim_list_bufs())
``` 
We can then name it `get_listed_buffers`

Reply:
Yep looks good! I was expecting this to change eventually to address #6.

Reply:
@ditsuke do you mind pushing the fix for this here (or making a suggestion we can commit) and adding a call to this in `autocommands`? I feel bad having my name on the git blame for your work (this was already your code above..!)

Reply:
Done with [a30aa4a](https://github.com/axkirillov/hbac.nvim/pull/5/commits/a30aa4ac7c96a7f345e046567fe79154a18554f9)